frontends/web/package.json

```diff
@@ -26,6 +26,7 @@
     "vitest": "^2.1.4"
   },
   "dependencies": {
+    "@playwright/test": "^1.55.0",
```

Comment:
tiny nit: This should be a dev dependency `npm install -D @playwright/test@1.55.0`